bundle_pyinstaller.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+######################################################################
+# repo-versions.sh
+# Copyright (C) 2014, 2015 LEAP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+set -e  # Exit immediately if a command exits with a non-zero status.
+
+REPOSITORIES="bitmask_client leap_pycommon soledad keymanager leap_mail"
+PACKAGES="leap_pycommon keymanager soledad/common soledad/client leap_mail bitmask_client"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+REPOS_ROOT="$SCRIPT_DIR/repositories"  # Root path for all the needed repositories
+VENV_DIR="$SCRIPT_DIR/bitmask.venv"  # Root path for all the needed repositories
+
+# Escape codes - set colors
+esc=`echo -en "\033"`
+cc_green="${esc}[0;32m"
+cc_yellow="${esc}[0;33m"
+cc_red="${esc}[0;31m"
+cc_normal=`echo -en "${esc}[m\017"`
+
+mkdir -p $REPOS_ROOT
+
+create_venv() {
+    status="creating virtualenv"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+    virtualenv $VENV_DIR && source $VENV_DIR/bin/activate
```

Comment:
I love virtualenvwrapper but, on scripts, I prefer to handle virtualenvs directly since the ammounts/complexity of steps is kind of the same, that way we don't depend on having an extra package installed and we keep the venv folder along the other stuff created for the script to run.

Why should we use venvwrapper on scripts?